### ISSUE TITLE
Font settings for the text inside the popup frames

### ISSUE THREAD

**Issue by Ellypse** [Ellypse/Storyline#5]
The text inside the objectives and reward popups are missing from the text settings. We want them to be customizable too, so people can pick a font and a size they are more comfortable with.
